samples/nested-pipeline/compose.py

```diff
@@ -95,6 +97,12 @@ def __init__(self, name, url):
   name='Download and Save Most Frequent',
   description='Download and Get Most Frequent Word and Save to GCS'
 )
-def download_save_most_frequent_word(url: str, outputpath: str):
+def download_save_most_frequent_word(
+                                url="gs://ml-pipeline-playground/shakespeare1.txt", 
+                                outputpath="res.txt"):
```

Comment:
we should keep the type checking while having default variables. e.g.
```suggestion
                                outputpath: str ="res.txt"):
```

Reply:
thanks.